src/ui/page/settings/view.jsx

```diff
@@ -429,24 +476,42 @@ class SettingsPage extends React.PureComponent<Props, State> {
                   )}
                 />
 
-                {
+                <FormField
+                  name="language_select"
+                  type="select"
+                  label={__('Language')}
+                  onChange={this.onLanguageChange}
+                  value={currentLanguage}
+                  helper={__(
+                    'Multi-language support is brand new and incomplete. Switching your language may have unintended consequences.'
+                  )}
+                >
+                  {Object.keys(languages).map(language => (
+                    <option key={language} value={language}>
+                      {languages[language]}
+                    </option>
+                  ))}
+                </FormField>
+              </Form>
+              <Form>
+                <fieldset-section>
                   <FormField
-                    name="language_select"
+                    name="max_connections"
                     type="select"
-                    label={__('Language')}
-                    onChange={this.onLanguageChange}
-                    value={currentLanguage}
-                    helper={__(
-                      'Multi-language support is brand new and incomplete. Switching your language may have unintended consequences.'
-                    )}
+                    label={__('Max Connections')}
+                    helper={__('More connections, like, do stuff dude')}
```

Comment:
@tzarebczan Can you add some help text for this setting?

Reply:
Will add it today.